src/routes/index.js

```diff
@@ -0,0 +1,20 @@
+var express = require('express');
+var router = express.Router();
+
+const { Pool } = require('pg');
+const pool = new Pool({
+  connectionString: process.env.DATABASE_URL
+});
+
+var sql_query = 'SELECT * FROM coap_post';
+
+/* GET home page. */
+router.get('/', function(req, res, next) {
+  pool.query(sql_query, (err, data) => {
+		res.render('index', { 
+			data: data.rows 
```

Comment:
`data` may be null which will result in failure to display page. Change line 15 to perhaps
`data: (data ? data.rows : [])`
to account for that

Reply:
Thanks for the suggestion. I have implemented this fix.

![image](https://user-images.githubusercontent.com/13582874/67159814-65bead00-f37c-11e9-921f-4c5316d96b04.png)